service/controller/v13/encrypter/kms/spec.go

```diff
@@ -0,0 +1,5 @@
+package kms
+
+const (
+	pendingDeletionWindow = 7
```

Comment:
It probably came from the existing code but could we have a comment to say this is 7 days?

Reply:
Sure, done.